### DISCUSSION TITLE
SSL support for prometheus query runner

### DISCUSSION THREAD

**Discussion by fabrei** [getredash/redash#6655] [Ideas]
Hi,

I want to share my implementation for ssl support for prometheus query runner: https://github.com/fabrei/redash/tree/ssl-support-for-prometheus

I wanted to create a new issue, but the template says, that I do have to create a new discussion for new features. So, here it is :) and a few questions: Can I create a pull request without a related issue? Or can I create an issue, new pull request and refer to it?

Kind regards
Fabian

**Comment by masayuki038** [comment]
Hi @fabrei,
I think you can create a PR for this without issue since you have explained your updates here.

**Reply by fabrei** [reply]
Thanks for your answer. I created a [PR](https://github.com/getredash/redash/pull/6657).
